flask_project/campaign_manager/templates/new_base.html

```diff
@@ -208,6 +211,7 @@ <h1 class="page-header">{% block title %}{% endblock %}</h1>
                     map.setView([0, 0], 1);
                 }
             }
+            profile_updated('');
```

Comment:
Hi @meomancer there is already function for this, see loginSuccess and logoutSuccess in new_base.html. You can use that, maybe add loginError() if login failed.